libs/ui/src/components/molecules/DataIndicator/DataIndicator.tsx

```diff
@@ -0,0 +1,36 @@
+import { DataMd, DataXl, ParMd, ParSm, Tooltip } from '../../atoms';
+import { DataIndicatorContainer } from './DataIndicator.styles';
+
+type DataIndicatorProps = {
+  label: string;
+  data: string;
+  size?: 'lg' | 'sm';
+  info?: string;
+};
+
+export const DataIndicator = ({
+  label,
+  data,
+  size = 'lg',
+  info,
+}: DataIndicatorProps) => {
+  if (size === 'sm') {
+    return (
+      <DataIndicatorContainer>
+        <ParMd style={{ marginBottom: '0.5rem' }}>
```

Comment:
Let's try to stay away from inline styles if possible. Makes it harder for other devs to add their own styles later on. 

You can create create a class and target that class inside or use IndicatorMd.